File: location_score/v1/get/Python SDK/location_score.py
# Install the Python library from https://pypi.org/project/amadeus
from ast import keyword
from amadeus import Client, ResponseError

amadeus = Client(
)

try:
    '''
    What are the location scores for the given coordinates?
    '''
    response = amadeus.location.analytics.category_rated_areas.get(latitude=41.397158, longitude=2.160873)
    print(response.data)
except ResponseError as error:
    raise error

File: location_score/v1/get/Node SDK/location_score.js
const Amadeus = require("amadeus");

const amadeus = new Amadeus();

async function main() {
  try {
    //What are the location scores for the given coordinates?
    const response = await amadeus.location.analytics.categoryRatedAreas.get({
      latitude: 41.397158,
      longitude: 2.160873,
    });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();
